modules/govuk_splunk/manifests/init.pp

```diff
@@ -0,0 +1,125 @@
+# == Class: govuk_splunk
+#
+# Install, configure and run Splunk universal forwarder
+#
+# === Parameters:
+#
+# [*gds_servers*]
+#   string of comma delimited "ipaddress:portnumber" of GDS Splunk cloud servers
+#
+# [*gds_server_cert*]
+#   Private certificate and key of forwarder to contact GDS Splunk cloud
+#
+# [*gds_root_ca_cert*]
+#   certificate of certificate authority of GDS Splunk cloud
+#
+# [*gds_password*]
+#   password to use for GDS Splunk cloud
+#
+# [*gds_cname*]
+#   common name of GDS Splunk cloud
+#
+# [*cyber_servers*]
+#   string of comma delimited "ipaddress:portnumber" of Cyber Splunk cloud
+#   servers
+#
+# [*cyber_server_cert*]
+#   certificate of Cyber Splunk cloud
+#
+# [*cyber_root_ca_cert*]
+#   certificate of certificate authority of Cyber Splunk cloud
+#
+# [*cyber_cname*]
+#   common name of Cyber Splunk cloud
+#
+#
+class govuk_splunk(
+  $gds_servers,
+  $gds_server_cert,
+  $gds_root_ca_cert,
+  $gds_password,
+  $gds_cname,
+  $cyber_servers,
+  $cyber_server_cert,
+  $cyber_root_ca_cert,
+  $cyber_cname,
+) {
+
+  include govuk_splunk::repo
```

Comment:
should this be: include govuk_splunk::repos

Reply:
good spot